hologram-minecraft-server.pkg.toml

```diff
@@ -20,9 +20,154 @@ group = "users"
+    command="/home/minecraft/bin/minecraftServerLauncher.sh",no-port-forwarding,no-X11-forwarding,no-agent-forwarding,no-pty ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAACAQC5SglxtK6N9Gfqcly/k5ozXSw4YIcmaWdsXnO40cEZWttTU0uuGxjhJQ2uppW7isHdgv9klQnJ8CE/tJ59i+JC8Hw3lZKYjO4MGEYBAWiEdh6fkszAmE7uDbFz4BpXuLd5f51fB4KlBUxvfq5ty5LSy6lCBUdn9UTO7NUTC/Bi7te3oxPCppHUOz5F/SmKyqrI/lCjhDXBIPUu1JSC38483FWWeZO/YPnoxtEbh9xPcKNeklUIkb03+zFRBsEmpLzSMBaIKDu884HZPK7jVLdthztKKFVLn/n1a6qk64mVm/QG1C+pJV93k+PdhWEXq3oHltYDDgCkaT8Ll//2XacAIP2CE7NTqjrCuNrq6u0yE7xjKj9PhpL1/jl5/4M173bxMv/x8ktK1iIKH0L+VfSj8JzhxIZVF7c9FQm9v4vLk0kcDyQKgnImfkotJjnS1y1HdseIFVogJb1lIz2NDdzB0/nRNZMky4yMTKZyx8vhA+6W9elFu2ofB8g7ONvDaKiMR7dBWE6ScB0tCvmJdygzlHqUzTiMKUUwZ+4EbNkYPchmFOs9COMkUzZ+BvmH/WM2LpGS7fJNXc7ygScrvZaNOxHm3ZEQHWCn3cOE9wBzOKwATimOVRc4f870ZHDyHkMCQVQwfQfKjrlwFIg+rFKWQ7qB1EzM3JxtovBYTIHHcw== stefan@krikkit
+    command="/home/minecraft/bin/minecraftServerLauncher.sh",no-port-forwarding,no-X11-forwarding,no-agent-forwarding,no-pty ssh-rsa AAAAB3NzaC1yc2EAAAABIwAAAgEAtD1Wf7vCrS8iQwDS3D+qt8WS2hOdLGUTz8XkjX4vDs5wBvcwvvUXnChSMJNYLQFuSw49XlasfGaDeZsPX+CUH/hozQF4y9TIy+/RinaA0rWFiEok6xB3trmcTDYMi43PxZqBqy5HIK0mhKEbCRcp9FNB5qhLKxOphARfXFtGPo2BulLpZpnTMFE6jJrWylcXOkb3C4+A/e71ztO8hwYemPWoYl0lqJMtPS1KoifjIpEWW5JkcvmS2MbXBAfKf/97YtXDBiLh4WRA8BsKM8/V0ZxNOpeCwxVs9lQMCkBzl7374A4OjAEbh7ie3BDr9IEYH4ofH+aAaO05IUsbCZvGn55dhCt+T1L7a9jDd6RsUJwazOHalqCXqN/4imEzhSFYE6B5QdbhqHL6+mhKG3SenjSWv49hvEojU/7XohLPvBgnvupTNktjDTBG+fUfuL2XbJRZ91d6tYZkjmlmNJox6dNE4WzHLd73WywUW3s7OxWIaz8vBvnIlRxXVSbG/Irxq7YPx7+L/rFsj55cchD2FEbgtBfbNiG88/49WyK3M+5yzX7fTdNPVMeNjEYA1A6+eqU0LauOejTA9Zp7ernBjl40i/lXSectBipZFUL7azRSE7RzY53Y7CeLS68HrmVos3mSO0rMU/Fk11xHodImgH/KbjNiIeArAq6uYRi7eRk= apophis@schlepptop
+    command="/home/minecraft/bin/minecraftServerLauncher.sh",no-port-forwarding,no-X11-forwarding,no-agent-forwarding,no-pty ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIBuoAQ9uQayjVuvFwnfV+pXrauYIvge6DGk9gdapIhoO apophis@toruk
+    command="/home/minecraft/bin/minecraftServerLauncher.sh",no-port-forwarding,no-X11-forwarding,no-agent-forwarding,no-pty ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQCs0PiLBRaS63NBDLYhXDzIvKT2a6rHbiosOiKIjjiOpn5LLZZPgaOfghp3z38ZwiUC5b3peL86TFC905zYU7KXXAoOGRVAT7dn+emCPcGn6164K2nDPGA1GvZP7gS2gUhvrxmDw66ilt32WWTyf4Xv5xwyOO8S0a98jc+MAib9ksETciea3XSAVswKUCG3SAR1KIru7qNPaPuGxu6QWQEjZgjkHTZ0IL7v82cNm/XxtTKtJDa4LRegxNUJBYOuz36C8kx6mlM3Y4K+d5eevOJKKjlldr2NWj+3GJbpV2ItO6fed/MzMCJLep61l5Fvr4RHWFZhHaPJ7QkUrqBnX4Kn felix
+'''
+
+[[file]]
+path = "/home/minecraft/bin/minecraftServerLauncher.sh"
+content = '''
```

Comment:
Can you move this into the `src/` directory, and import it with `contentFrom = "src/filename.sh"`? Makes for better syntax highlighting.